examples/labs/orientation/main.cpp

```diff
@@ -27,26 +26,25 @@
  * POSSIBILITY OF SUCH DAMAGE.
  **/
 
-
-#include <QtGui>
+#include <QApplication>
 #include "orientationview.h"
 #include "orientation.h"
 
 int main(int argc, char **argv)
 {
     QApplication app(argc, argv);
-    OrientationView view;
 
+    OrientationView view;   // QWidget that embeds a Qt3D window
     Orientation orientation;
     orientation.start();
 
     QObject::connect(&orientation,
-            SIGNAL(orientation(qreal,qreal,qreal,qreal,qreal,qreal,qreal)),
-            &view,
-            SLOT(orientation(qreal,qreal,qreal,qreal,qreal,qreal,qreal)));
+        &Orientation::orientation,           // new function-pointer syntax
```

Comment:
This is a good change, but the comment is more for explaining the PR and not the code:

```suggestion
        &Orientation::orientation,
```